requirements_test_all.txt

```diff
@@ -17,6 +17,7 @@ pytest-sugar==0.9.2
 pytest-timeout==1.3.3
 pytest==4.4.0
 requests_mock==1.5.2
+aioswitcher==2019.3.21
```

Comment:
To get your lib available in tests, add your package to the `TEST_REQUIREMENTS` const in `script/gen_requirements_all.py`

Reply:
My bad.
I wasn't aware of the `TEST_REQUIREMENTS` constant.
I was under the impression that the `requirements_test.txt` file is allegedly the place to add the package.
The addition to this `requirements_test_all.txt` was actually not done manually but by using the `script/gen_requirements_all.py`.

I'll fix this, thank you for the clarification.

Reply:
Fixed in this [commit](https://github.com/home-assistant/home-assistant/pull/22325/commits/c18e74222cd98c988bd687fd00994a928c8f1ca8).